.github/workflows/docs.yml

```diff
@@ -0,0 +1,40 @@
+
+name: Documentation
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    container:
+      image: 'archlinux:latest'
+
+    steps:
+      - name: Install dependencies
+        run: |
+          # assume yes and do not ask for confirmation when installing deps using pacman
+          yes | pacman -Syu base-devel sudo git wget curl python-sphinx rsync --noconfirm
```

Comment:
`--noconfirm` alone should be enough, why do you also need the pipe with yes?

Reply:
https://unix.stackexchange.com/a/293537